src/main.cc

```diff
@@ -619,30 +619,32 @@ print_version(void)
 static void
 print_help(FILE *stream, const char *program)
 {
-	fputs("Tarantool - a Lua application server", stream);
-	fputs("", stream);
-	fprintf(stream, "Usage: %s script.lua [OPTIONS] [SCRIPT [ARGS]]\n",
-		program);
-	fputs("", stream);
-	fputs("All command line options are passed to the interpreted script.",
-	      stream);
-	fputs("When no script name is provided, the server responds to:",
-	      stream);
-	fputs("  -h, --help\t\t\tdisplay this help and exit", stream);
-	fputs("  -v, --version\t\t\tprint program version and exit", stream);
-	fputs("  -e EXPR\t\t\texecute string 'EXPR'", stream);
-	fputs("  -l NAME\t\t\trequire library 'NAME'", stream);
-	fputs("  -j cmd\t\t\tperform LuaJIT control command", stream);
-	fputs("  -b ...\t\t\tsave or list bytecode", stream);
-	fputs("  -d\t\t\t\tactivate debugging session for 'SCRIPT'", stream);
-	fputs("  -i\t\t\t\tenter interactive mode after executing 'SCRIPT'",
-	      stream);
-	fputs("  --\t\t\t\tstop handling options", stream);
-	fputs("  -\t\t\t\texecute stdin and stop handling options", stream);
-	fputs("", stream);
-	fputs("Please visit project home page at https://tarantool.org", stream);
-	fputs("to see online documentation, submit bugs or contribute a patch.",
-	      stream);
+	static const char help_msg[] = "Tarantool Server %s\n\n"
+		"Connect to an instance:\n\n"
+		"  tt connect <uri>\n\n"
+		"Execute Lua script with bundled LuaJIT:\n\n"
+		"  %s old-style-script.lua\n\n"
```

Comment:
`[ARGS]` is missed, so `--` below looks like it is redundant.

Reply:
Added, thanks!

Reply:
Hm. There is the following scenario:

```
%s old-style-script.lua [ARGS]
```

It has no `[OPTIONS]`, but, it seems, it should.

And there is the separate block:

```
Usage:
%s [OPTIONS] [SCRIPT [ARGS]]
```

Which duplicates the old-style-script scenario, but describes the CLI arguments correctly.

I think we should remove the 'Usage:' block and incorporate `[OPTIONS]` into the old-style-script block.

Reply:
Added `[OPTIONS]` for "old style", since some options definitely can be used in this case. All other changes will be made in scope of another PR.